pkg/sink/sink.go

```diff
@@ -111,6 +123,7 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 		log.Errorf("Error reading event body: %s", err)
 		r.recordCountMetrics(failTag)
 		response.WriteHeader(http.StatusInternalServerError)
+		r.emitEvents(r.ContextForEvents, el, err)
```

Comment:
I think this should be co-ordinated with #1259 as well - can we have a single `r.EmitEvents` that sends both K8sEvents and CloudEvents (if enabled).

Reply:
Yup i agree but i think we can do that once PRs merge